Themis/Views/AppearanceSettings/AppearanceSettingsView.swift

```diff
@@ -1,19 +1,21 @@
 import SwiftUI
 
 struct AppearanceSettingsView: View {
-    @StateObject var model = AssessmentViewModel.mock
+    @ObservedObject var vm: CodeEditorViewModel
     @Binding var showSettings: Bool
+
     var body: some View {
         VStack {
-            EditorFontSizeStepperView(model: model)
+            EditorFontSizeStepperView(vm: vm)
             Spacer()
         }
     }
 }
-struct AppearanceSettingsView_Previews: PreviewProvider {
+
+/*struct AppearanceSettingsView_Previews: PreviewProvider {
     @State static var showSettings: Bool = true
     static var previews: some View {
         AppearanceSettingsView(showSettings: $showSettings)
             .previewInterfaceOrientation(.landscapeLeft)
     }
-}
+}*/
```

Comment:
Why is this commented? I would suggest to either make it work or if it is useless just remove it completely. We have Git so nothing will get lost :-)

Reply:
I don't use the preview so I was just to lazy to instantiate the vm. Will just delete it then.